attendance/index.html

```diff
@@ -0,0 +1,240 @@
+            <label for='meetingEndTime'>End Time</label>
+            <input class='timepicker' id='meetingEndTime'>
+        </div>
+
+        <div class="my-5">
+            <label for='meetingDescription'>Meeting Description</label>
+            <textarea 
+                maxlength=1000 
+                required=true
+                class='w-100'
+                id='meetingDescription'
+                placeholder='(Optional) A short description of what happened at the meeting.'></textarea>
+        </div>
+
+        <div class="my-5">
+            <label for='uniqname'>Enter uniqnames of people at meeting. Hit enter to add person.</label>
+            <br/>
+            <input type='text' id='uniqname'>
+            <br/>
+            <div id='uniqname-buttons'>
+
+            </div>
+        </div>
+
+        <button type="button" class="btn btn-primary my-3" data-toggle="modal" data-target="#confirmMeeting" id="confirm">Confirm And Submit</button>
+    </form>
+            
+    <!-- Modal -->
+    <div class="modal fade" id="confirmMeeting" tabindex="-1" role="dialog" aria-labelledby="confirmMeetingTitle" aria-hidden="true">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+        <div class="modal-header">
+            <h5 class="modal-title" id="confirmMeetingTitle">Confirm Submission</h5>
+            <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+            <span aria-hidden="true">&times;</span>
+            </button>
+        </div>
+        <div class="modal-body">
+            <div id="modalError" style="color:red"></div>
+            <h2>Meeting Details</h2>
+            <p id="modalMeetingType">Meeting Type: </p>
+            <p id="modalStartTime">Start Time: </p>
+            <p id="modalEndTime">End Time: </p>
+            <p id="modalDescription">Description: </p>
+            <h2>Meeting Attendees</h2>
+            <div id="modalAttendees">
+
+            </div>
+        </div>
+        <div class="modal-footer">
+            <button type="button" class="btn btn-secondary" id="closeModal" data-dismiss="modal">Close</button>
+            <button type="button" class="btn btn-primary submit-meeting" id="submit" data-dismiss="modal">Create Meeting</button>
+        </div>
+        </div>
+    </div>
+    </div>
+
+    
+    {% include footer.html %}
+    {% include scripts.html %}
+    <script src='//cdnjs.cloudflare.com/ajax/libs/timepicker/1.3.5/jquery.timepicker.min.js'></script>
+    <script>
+        let convertMeetingTime = (time) => {
+            if(time.includes('AM') || time.includes('12')) {
+                return time.slice(0, 5);
+            } else {
+                let hour = parseInt(time.slice(0, 3));
+                return `${(hour + 12)}:${time.slice(2, 4)}`;
+            }
+        }
+
+        let validateTime = (start, end) =>{
+            return parseInt(start.replace(':', '')) < parseInt(end.replace(':', ''));
+        } 
```

Comment:
Do we have a preference on ```let``` vs ```const``` on functions? 
These two are one way but within the document ready function everything is ```const```

Reply:
i'll go with const